### PR TITLE
[docs] Enhance documentation with Quick Start Guide and Detailed System Requirements

### PR DESCRIPTION
**Pull request by AleksZimin**
## Description

This PR enriches our documentation by introducing a 'Quick Start' section, designed to help new users easily get up and running with our solution. We have also expanded the 'System Requirements and Recommendations' section, providing users with a comprehensive understanding of the hardware and software prerequisites for optimal performance.

## Why do we need it, and what problem does it solve?

**User Onboarding:** The 'Quick Start' section significantly lowers the entry barrier for new users, offering step-by-step guidance for setting up our solution quickly.
**Informed Setup:** By detailing system requirements and recommendations more thoroughly, users can prepare their environment more effectively, ensuring compatibility and optimal performance.

## What is the expected result?

- Users, especially those new to our solution, will find it easier to start using our product, thanks to clear, straightforward setup instructions.
- The detailed system requirements will help users configure their systems appropriately, leading to better performance and fewer issues down the line.
- Overall, these documentation improvements will contribute to a smoother onboarding experience and greater satisfaction among our user base.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: docs/FAQ_RU.md [hunk 1]
```diff
@@ -13,47 +13,47 @@ description: Диагностика проблем LINSTOR. Когда след
 
 ## Когда следует использовать LVM, а когда LVMThin?
 
-- LVM проще и обладает производительностью, сравнимой с производительностью накопителя;
-- LVMThin позволяет использовать snapshot'ы и overprovisioning, но существенно медленнее.
+- LVM проще и обладает высокой производительностью, сравнимой с производительностью накопителя;
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:16]:
```suggestion
- LVM проще и обладает высокой производительностью, сравнимой с производительностью накопителя, но не позволяет использовать snapshot'ы
```

File: docs/FAQ_RU.md [hunk 2]
```diff
@@ -13,47 +13,47 @@ description: Диагностика проблем LINSTOR. Когда след
...
+  > **Внимание!** Эта информация отражает состояние *сырого* пространства для каждого узла в кластере. Если создаются тома в двух репликах, то эти две реплики обязательно должны целиком поместиться на двух узлах вашего кластера.
 
 ## Как назначить StorageClass по умолчанию?
 
 В соответствующем пользовательском ресурсе [DRBDStorageClass](./cr.html#drbdstorageclass) в поле `spec.isDefault` указать `true`.  
 
 ## Как добавить существующую LVM Volume Group или LVMThin-пул?
 
-1. Вручную назначить на Volume Group тег `storage.deckhouse.io/enabled=true`:
+1. Вручную назначьте на Volume Group тег `storage.deckhouse.io/enabled=true`:
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:44]:
```suggestion
1. Вручную назначьте на Volume Group LVM-тег `storage.deckhouse.io/enabled=true`:
```

File: docs/FAQ_RU.md [hunk 3]
```diff
@@ -13,47 +13,47 @@ description: Диагностика проблем LINSTOR. Когда след
...
    vgchange myvg-0 --add-tag storage.deckhouse.io/enabled=true
    ```
 
-2. Данный VG будет автоматически обнаружен и в кластере для него будет создан соответствующий ресурс `LVMVolumeGroup`.
+   Данный VG будет автоматически обнаружен и в кластере для него будет создан соответствующий ресурс `LVMVolumeGroup`.
 
-3. Этот ресурс можно указать в параметрах [DRBDStoragePool](./cr.html#drbdstoragepool) в поле `spec.lvmVolumeGroups[].name` (для LVMThin-пула необходимо дополнительно указать имя в `spec.lvmVolumeGroups[].thinPoolName`).
+2. Полученный ресурс укажите в параметрах [DRBDStoragePool](./cr.html#drbdstoragepool) в поле `spec.lvmVolumeGroups[].name` (для LVMThin-пула необходимо дополнительно указать имя в `spec.lvmVolumeGroups[].thinPoolName`).
 
 ## Как выгнать ресурсы с узла?
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:54]:
```suggestion
## Как выгнать DRBD-ресурсы с узла?
```

File: docs/FAQ_RU.md [hunk 4]
```diff
@@ -69,21 +69,21 @@ description: Диагностика проблем LINSTOR. Когда след
...
 
-* Убедитесь, что все поды в пространстве имен `d8-sds-drbd` находятся в состоянии *Running*:
+3. Убедитесь, что все поды в пространстве имен `d8-sds-drbd` находятся в состоянии *Running*:
 
   ```shell
   kubectl -n d8-sds-drbd get pods | grep -v Running
   ```
 
-### Выгнать ресурсы с узла без удаления его из LINSTOR и Kubernetes
+### Можно ли удалить ресурсы с узла в системах управления кластером LINSTOR и Kubernetes, без удаления самого узла?
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:84]:
```suggestion
### Можно ли удалить DRBD-ресурсы с узла, без удаления самого узла из LINSTOR и Kubernetes?
```

File: docs/FAQ_RU.md [hunk 5]
```diff
@@ -124,40 +124,40 @@ linstor node list -s AutoplaceTarget
...
 journalctl -fu bashible
 ```
 
-Наиболее вероятные причины, почему он не может загрузить модуль ядра:
+Наиболее вероятные причины, почему под не может загрузить модуль ядра:
 
-- Возможно, у вас уже загружена in-tree-версия модуля DRBDv8, тогда как модуль требует DRBDv9.
-  Проверить версию загруженного модуля: `cat /proc/drbd`. Если файл отсутствует, значит, модуль не загружен и проблема не в этом.
+- Возможно, загружена in-tree-версия модуля DRBDv8, тогда как модуль требует DRBDv9.
+  Проверить версию загруженного модуля можно файлом: `cat /proc/drbd`. Если файл отсутствует, значит, модуль не загружен и проблема не в этом.
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:143]:
```suggestion
  Проверить версию загруженного модуля можно командой: `cat /proc/drbd`. Если файл отсутствует, значит, модуль не загружен и проблема не в этом.
```

File: docs/FAQ_RU.md [hunk 6]
```diff
@@ -124,40 +124,40 @@ linstor node list -s AutoplaceTarget
...
-Если под завис на стадии `ContainerCreating`, а в выводе `kubectl describe pod` есть ошибки следующего вида, то значит устройство все еще смонтировано на одном из других узлов:
+Если под завис на стадии `ContainerCreating`, а в выводе `kubectl describe pod` присутствуют ошибки, как представлено ниже, значит устройство смонтировано на одном из других узлов:
 
 ```text
 rpc error: code = Internal desc = NodePublishVolume failed for pvc-b3e51b8a-9733-4d9a-bf34-84e0fee3168d: checking
 for exclusive open failed: wrong medium type, check device health
 ```
 
-Проверить это можно с помощью следующей команды:
+Проверьте вывод с помощью следующей команды:
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:160]:
```suggestion
Проверьте, где используется устройство, с помощью следующей команды:
```

File: docs/FAQ_RU.md [hunk 7]
```diff
@@ -202,9 +203,44 @@ alias linstor='kubectl -n d8-sds-drbd exec -ti deploy/linstor-controller -- lins
 linstor --help
 ```
 
+## Поды компонентов sds-drbd не создаются на нужной мне ноде.
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:206]:
```suggestion
## Служебные поды компонентов sds-drbd не создаются на нужной мне ноде.
```

File: docs/FAQ_RU.md [hunk 8]
```diff
@@ -250,15 +286,15 @@ spec:
...
 
 ```shell
 kubectl get moduleconfig sds-node-configurator
 ```
 
-- Создать ресурс `ModuleConfig` для `sds-drbd`.
+6. Создайте ресурс `ModuleConfig` для `sds-drbd`.
 
-> **Внимание!** Если в настройках модуля `sds-drbd` вы не укажете параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым.
+> **Внимание!** Если в настройках модуля `sds-drbd` не будет указан параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым.
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:297]:
```suggestion
> **Внимание!** Если в настройках модуля `sds-drbd` не будет указан параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым и все узлы кластера будут считаться как узлы для хранения данных.
```

File: docs/FAQ_RU.md [hunk 9]
```diff
@@ -328,3 +364,14 @@ StorageClass'ы в данном модуле управляются через 
 ### Миграция на DRBDStoragePool
 
 Ресурс `DRBDStoragePool` позволяет создавать `Storage Pool` в `LINSTOR`. Рекомендуется создать этот ресурс даже для уже существующих в LINSTOR `Storage Pool` и указать в этом ресурсе существующие `LVMVolumeGroup`. В этом случае контроллер увидит, что соответствующие `Storage Pool` созданы, и оставит их без изменений, а в поле `status.phase` созданного ресурса будет отображено значение `Created`. Подробнее про работу с ресурсами `LVMVolumeGroup` можно прочитать в документации модуля [sds-node-configurator](../../sds-node-configurator/stable/usage.html), а с ресурсами `DRBDStoragePool` [здесь](./usage.html).
+
+
+## Почему вы не рекомендуете использовать RAID?
```
Comment (AleksZimin) [inline docs/FAQ_RU.md:369]:
```suggestion
## Почему не рекомендуется использовать RAID для дисков, которые используются модулем `sds-drbd`?
```

File: docs/README_RU.md [hunk 10]
```diff
@@ -26,3 +23,236 @@ moduleStatus: experimental
...
+NAME                                           NODE       CONSUMABLE   SIZE      PATH
+dev-0a29d20f9640f3098934bca7325f3080d9b6ef74   worker-0   true         30Gi      /dev/vdd
+dev-457ab28d75c6e9c0dfd50febaac785c838f9bf97   worker-0   false        20Gi      /dev/vde
+dev-49ff548dfacba65d951d2886c6ffc25d345bb548   worker-1   true         35Gi      /dev/vde
+dev-75d455a9c59858cf2b571d196ffd9883f1349d2e   worker-2   true         35Gi      /dev/vdd
+dev-ecf886f85638ee6af563e5f848d2878abae1dcfd   worker-0   true         5Gi       /dev/vdb
+```
+
+
+- Создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) узлы `worker-0`:
```
Comment (kirkonru) [inline docs/README_RU.md:101]:
```suggestion
- Создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) для узла `worker-0`:
```

File: docs/README_RU.md [hunk 11]
```diff
@@ -26,3 +23,236 @@ moduleStatus: experimental
...
+
+- Дождаться, когда созданный ресурс `LVMVolumeGroup` перейдет в состояние `Operational`:
+
+```shell
+kubectl get lvg vg-1-on-worker-0 -w
+```
+
+- Если ресурс перешел в состояние `Operational`, то это значит, что на узле `worker-0` из блочных устройств `/dev/vdd` и `/dev/vdb` была создана LVM VG с именем `vg-1`.
+
+- Далее создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) узлы `worker-1`:
```
Comment (kirkonru) [inline docs/README_RU.md:126]:
```suggestion
- Далее создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) для узла `worker-1`:
```

File: docs/README_RU.md [hunk 12]
```diff
@@ -26,3 +23,236 @@ moduleStatus: experimental
...
+
+- Дождаться, когда созданный ресурс `LVMVolumeGroup` перейдет в состояние `Operational`:
+
+```shell
+kubectl get lvg vg-1-on-worker-1 -w
+```
+
+- Если ресурс перешел в состояние `Operational`, то это значит, что на узле `worker-1` из блочного устройства `/dev/vde` была создана LVM VG с именем `vg-1`.
+
+- Далее создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) узлы `worker-2`:
```
Comment (kirkonru) [inline docs/README_RU.md:150]:
```suggestion
- Далее создать ресурс [LVMVolumeGroup](../../sds-node-configurator/stable/cr.html#lvmvolumegroup) для узла `worker-2`:
```

File: docs/FAQ_RU.md [hunk 13]
```diff
@@ -124,40 +124,40 @@ linstor node list -s AutoplaceTarget
...
+- Возможно, включен Secure Boot.
+  Так как модуль DRBD компилируется динамически для вашего ядра (аналог dkms), он не имеет цифровой подписи.
   На данный момент мы не поддерживаем работу модуля DRBD в конфигурации с Secure Boot.
 
 ### Под не может запуститься из-за ошибки `FailedMount`
 
 #### **Под завис на стадии `ContainerCreating`**
 
-Если под завис на стадии `ContainerCreating`, а в выводе `kubectl describe pod` есть ошибки следующего вида, то значит устройство все еще смонтировано на одном из других узлов:
+Если под завис на стадии `ContainerCreating`, а в выводе `kubectl describe pod` присутствуют ошибки, как представлено ниже, значит устройство смонтировано на одном из других узлов:
```
Comment (kirkonru) [inline docs/FAQ_RU.md:153]:
```suggestion
Если под завис на стадии `ContainerCreating`, а в выводе `kubectl describe pod` присутствуют ошибки вроде той, что представлена ниже, значит устройство смонтировано на одном из других узлов:
```

File: docs/FAQ_RU.md [hunk 14]
```diff
@@ -250,15 +286,15 @@ spec:
...
 
 ```shell
 kubectl get moduleconfig sds-node-configurator
 ```
 
-- Создать ресурс `ModuleConfig` для `sds-drbd`.
+6. Создайте ресурс `ModuleConfig` для `sds-drbd`.
 
-> **Внимание!** Если в настройках модуля `sds-drbd` вы не укажете параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым.
+> **Внимание!** Если в настройках модуля `sds-drbd` не будет указан параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым и все узлы кластера будут считаться как узлы для хранения данных.
```
Comment (kirkonru) [inline docs/FAQ_RU.md:297]:
```suggestion
> **Внимание!** Если в настройках модуля `sds-drbd` не будет указан параметр `settings.dataNodes.nodeSelector`, то значение для этого параметра при установке модуля `sds-drbd` будет взято из модуля `linstor`. Если этот параметр не указан и там, то только в этом случае он останется пустым и все узлы кластера будут считаться узлами для хранения данных.
```

File: docs/FAQ_RU.md [hunk 15]
```diff
@@ -272,39 +308,39 @@ spec:
...
-- Измените алиас к команде `linstor` и проверьте ресурсы `LINSTOR`:
+10. Измените алиас к команде `linstor` и проверьте ресурсы `LINSTOR`:
 
 ```shell
 alias linstor='kubectl -n d8-sds-drbd exec -ti deploy/linstor-controller -- linstor'
 linstor resource list --faulty
 ```
 
-Если "плохие" ресурсы не появились, значит миграция выполнена успешно.
+Если “плохие” ресурсы не обнаружены, значит, миграция была успешной.
```
Comment (kirkonru) [inline docs/FAQ_RU.md:337]:
```suggestion
Если “плохие” ресурсы не обнаружены, значит миграция была успешной.
```

File: docs/FAQ_RU.md [hunk 16]
```diff
@@ -328,3 +364,16 @@ StorageClass'ы в данном модуле управляются через 
...
+
+DRBD с количеством реплик больше 1 предоставляет по факту сетевой RAID. Использование RAID локально может быть неэффективным, так как:
+
+- В несколько раз увеличивает оверхед по используемому пространству в случае использования RAID с избыточностью. Пример: используется `DBRDStorageClass` с `replication`, выставленном в `ConsistencyAndAvailability`. При таких настройках DRBD будет сохранять данные в трех репликах (по одной реплике на три разных хоста). Если на этих хостах будет использоваться RAID1, то для хранения 1 Гб данных потребуется суммарно 6 Гб места на дисках. RAID с избыточностью есть смысл использовать для упрощения обслуживания серверов в том случае, когда цена хранения не имеет значения. RAID1 в таком случае позволит менять диски на серверах без необходимости перемещения реплик данных с "проблемного" диска.
+
+- В случае RAID0 прирост производительности будет незаметен, тк репликация данных будет осуществляться по сети и узким местом с высокой вероятностью будет именно сеть. К тому же, уменьшение надежности хранилища на хосте потенциально будет приводить к недоступности данных, тк в DRBD переключение со сломавшейся реплики на здоровую происходит не мгновенно.
```
Comment (kirkonru) [inline docs/FAQ_RU.md:375]:
```suggestion
- В случае RAID0 прирост производительности будет незаметен, т. к. репликация данных будет осуществляться по сети и узким местом с высокой вероятностью будет именно сеть. Кроме того, уменьшение надежности хранилища на хосте потенциально будет приводить к недоступности данных, тк в DRBD переключение со сломавшейся реплики на здоровую происходит не мгновенно.
```

File: docs/README_RU.md [hunk 17]
```diff
@@ -26,3 +23,236 @@ moduleStatus: experimental
 
 Поддерживаются два режима — LVM и LVMThin.
 Каждый из них имеет свои достоинства и недостатки, подробнее о различиях читайте в [FAQ](./faq.html#когда-следует-использовать-lvm-а-когда-lvmthin).
+
+## Быстрый старт
+
+Все команды выполнять на машине, имеющей доступ к API Kubernetes с правами администратора.
```
Comment (kirkonru) [inline docs/README_RU.md:29]:
```suggestion
Все команды следует выполнять на машине, имеющей доступ к API Kubernetes с правами администратора.
```

File: docs/README_RU.md [hunk 18]
```diff
@@ -26,3 +23,236 @@ moduleStatus: experimental
...
+
+```
+
+- Дождаться, когда модуль перейдет в состояние `Ready`. На этом этапе НЕ нужно проверять поды в namespace `d8-sds-node-configurator`.
+
+```shell
+kubectl get mc sds-node-configurator -w
+```
+
+- Включить модуль `sds-drbd`. Возможные настройки модуля рекомендуем посмотреть в [конфигурации](./configuration.html). В примере ниже модуль запускается с настройками по умолчанию. Это приведет к тому, что на все узлы кластера будет установлен модуль ядра `DRBD`, зарегистрирован CSI драйвер, а так же запущены служебные поды компонентов `sds-drbd`.
```
Comment (kirkonru) [inline docs/README_RU.md:54]:
```suggestion
- Включить модуль `sds-drbd`. Возможные настройки модуля рекомендуем посмотреть в [конфигурации](./configuration.html). В примере ниже модуль запускается с настройками по умолчанию. Это приведет к тому, что на все узлы кластера будет установлен модуль ядра `DRBD`, зарегистрирован CSI драйвер, а также запущены служебные поды компонентов `sds-drbd`.
```

**Review by kirkonru — APPROVED** [review]
Did some minor edits, mostly to get rid of a typo we discussed in the Slack thread.

**Pull request merged**